### DISCUSSION TITLE
[question] how to hide only private activity

### DISCUSSION THREAD

**Discussion by jopemachine** [lowlighter/metrics#141] [Q&A] [answered]
### Question

Hi, Thanks for creating awesome lib :)

I've read I can hide my private activity by providing only public permission to github token in readme.

But I wonder if I could change only the items in `activity` to the public while leaving other items (like language, repositories..) private.

**Comment by lowlighter** [comment, marked as answer]
Hello!
Thanks 🙂 !

This feature is not yet available on `@latest` (v.3.3) but will be available in next release (v3.4) with this new option:
https://github.com/lowlighter/metrics/blob/a0eaef9450bde0adde17a67710d7f815c4c10a62/source/plugins/activity/metadata.yml#L56-L63

If you're using `@master` branch, you should be able to set `plugin_activity_visibility: public` to hide private activity, else you can just wait for next release to roll out 🙂

**Comment by Nixinova** [comment]
You may want to look into enabling Discussions on this repo to better organise questions like this one.
